pkg/provider/cloud/kubevirt/csi.go

```diff
@@ -0,0 +1,163 @@
+package kubevirt
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	restclient "k8s.io/client-go/rest"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	InfraStorageClassAnnotation = "kubevirt-initialization.k8c.io/initialize-sc"
+
+	csiServiceAccountNamespace = metav1.NamespaceDefault
```

Comment:
This is another issue, CSI SA should be created in the cluster namespace. I will create follow-up PRs to refactor KubeVirt a little bit to change this.